exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/entities/ColumnWithTransformTest.kt

```diff
@@ -0,0 +1,80 @@
+package org.jetbrains.exposed.sql.tests.shared.entities
+
+import org.jetbrains.exposed.dao.IntEntity
+import org.jetbrains.exposed.dao.IntEntityClass
+import org.jetbrains.exposed.dao.id.EntityID
+import org.jetbrains.exposed.dao.id.IntIdTable
+import org.jetbrains.exposed.sql.Op
+import org.jetbrains.exposed.sql.select
+import org.jetbrains.exposed.sql.tests.DatabaseTestsBase
+import org.jetbrains.exposed.sql.tests.shared.assertEquals
+import org.junit.Test
+
+object TransformTables {
+    object Transformations : IntIdTable() {
+        val value = varchar("value", 50)
+    }
+    object NullableTransformations: IntIdTable() {
+        val value = varchar("nullable", 50).nullable()
+    }
+    class Transformation(id: EntityID<Int>) : IntEntity(id) {
+        companion object : IntEntityClass<Transformation>(Transformations)
+        var value by Transformations.value.transform(
+            toColumn = { "transformed-$it" },
+            toReal = { it.replace("transformed-", "") }
+        )
+    }
+    class NullableTransformation(id: EntityID<Int>) : IntEntity(id) {
+        companion object : IntEntityClass<NullableTransformation>(NullableTransformations)
+        var value by NullableTransformations.value.transform(
+            toColumn = { "transformed-$it" },
+            toReal = { it?.replace("transformed-", "") }
+        )
+    }
+}
```

Comment:
What do you think of taking these out of `TransformTables` to make invoking them below less verbose?